hack/upgrade-test-harness/fixture/eck.go

```diff
@@ -0,0 +1,257 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package fixture
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/cli-runtime/pkg/resource"
+)
+
+const (
+	operatorNamespace      = "elastic-system"
+	operatorSTS            = "elastic-operator"
+	wantAPMServerNodes     = 1
+	wantElasticsearchNodes = 3
+	wantHealth             = "green"
+	wantKibanaNodes        = 1
+)
+
+// TestParam holds parameters for a test.
+type TestParam struct {
+	Name            string `json:"name"`
+	OperatorVersion string `json:"operatorVersion"`
```

Comment:
Not sure what the purpose of this attribute is as the actual operator version is determined by the YAML in `testdata` and this is not even used for logging

Reply:
Yes, it's a leftover from an earlier attempt. However, I kind of like the fact that it acts as documentation when looking at the config file, so I am inclined to leave it as is unless you feel very strongly about it.

Reply:
No not very strong at all.